src/api/user-recommendations/controllers/user-recommendations.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const NodeCache = require('node-cache');
+
+/**
+ * Read the documentation (https://strapi.io/documentation/developer-docs/latest/development/backend-customization.html#core-controllers)
+ * to customize this controller
+ */
+
+const myCache = new NodeCache({ stdTTL: 60 });
+
+module.exports = {
+  async recommend(ctx) {
+    let users = myCache.get('randomUsers');
+
+    // check if cache already has random users
+    if (users == undefined) {
+      const userCount = await strapi.entityService.count(
+        'plugin::users-permissions.user'
+      );
+
+      // gets 100 random user ids
```

Comment:
There are less than 100 users in staging.

Reply:
Could I add more users to staging? I was able to quickly make 150 users on my local environment.

Reply:
In staging we use Google for authentication, so each user needs a Google account.

Reply:
I think as the code is currently, it would still be able to provide recommendations even if there's less than 100 total users. The only issue is that there wouldn't be any randomness. Another way to test in staging would be if instead of 100 random users, I grab about half of the total users and rank them by similarity.